### DISCUSSION TITLE
Revisit passive HP drain for mania

### DISCUSSION THREAD

**Discussion by IceDynamix** [ppy/osu#21959] [Ideas] [answered]
Related issue https://github.com/ppy/osu/issues/7370

Passive HP drain is unheard of in the VSRG genre, with most games using either accumulating HP to pass (eg. SDVX Normal Clear, IIDX Easy/Normal Clear, Chunithm) or use the current model with no passive drain (SDVX Hard Clear, IIDX Hard/EX Clear, DDR/ITG). I do not currently know of any game that uses a model with passive HP drain, if you do know a game then please post it in the discussion.

Since there is both a desire from the developer's side to keep passive HP drain for mania ([1](https://github.com/ppy/osu/issues/7370#issuecomment-569241282), [2](https://twitter.com/ppy/status/1608811966533079040)) while also [a clear consensus that it should be removed](https://twitter.com/bojj__/status/1608662069511061504), I feel like there should be more discussion behind the intents of keeping HP drain and how to work around issues regarding it.

1. What does keeping passive HP try to change for the better? Why keep it in the first place? Do the reasons make it worth to consider keeping passive HP drain? (this is the most important point of discussion) 
2. All mania maps have HP values set without passive drain in mind, so maps might play out completely differently with unintended side effects (for example maps with large breaks). Is the goal to make HP passing easier, harder or equally hard compared to stable?
3. HP in mania is usually around 8-9, while its usually between 6-7 in other gamemodes, will this discrepency be addressed?
4. How exactly is HP going to be balanced in order to accomodate passive drain?

**Comment by peppy** [comment, marked as answer]
I've reopened the issue and changed the direction. https://github.com/ppy/osu/issues/7370

**Reply by IceDynamix** [reply]
This conclusion irks me a little bit.

I personally am _genuinely_ curious about the reasons considering passive HP drain and really open to the idea of keeping it. But if this is the way it's going to go, then it's just going to leave a bad taste in your mouth, in the sense that you're _only_ complying because there's massive backlash. The reason there is massive backlash is yes, mostly because people are simpleminded and want things to stay as they are, but also mainly because we _tried_ to see benefits to passive HP drain, but couldn't see any.

Maybe we're shortsighted and there are deeper reasons we can't understand yet. Maybe you have future plans that we don't know about. If you have genuine interest in keeping it, then I would love to hear the reasons. I would rather fight against reasons and come to a compromise, than have you "give up" with a passive-aggressive remark like that.

**Reply by IceDynamix** [reply]
My main goal of opening this discussion was to, well, have a discussion about this. I want to understand and I want to help you push your vision if it allows for it. There is no point if there is no discussion, we can only come to a proper conclusion once we hear both sides of the story.

**Reply by peppy** [reply]
Are you irked by the "passive-aggressive" part alone? If so, I can't see anything I said that looks to be passive-aggressive, so you may be reading that wrong.

At the end of the day, we are looking to create a game that is playable for the majority of users in the default state. The most common feedback we've seen from osu!mania players trying lazer is that they don't want drain. Rather than spending days discussing it (we've already done this), I am making the executive call of restoring things to how they were so we can move forward.

I understand your desire to understand where we are coming from and why we even considered adding drain, but at this point we have higher priorities and honestly just want to give users something they can move to from stable, so we can continue to make forward progress. We might reconsider adding other drain types in the future.

Does that addrese your concerns?

**Reply by IceDynamix** [reply]
For the passive-aggressive part I was referring to [this comment](https://github.com/ppy/osu/issues/7370#issuecomment-1375265315) specifically, I should have clarified that. I understand why you made the decision and do agree that it's the best path to take as of now. If you have discussed this in the past before, it would be interesting to see the discussion on this if there is a log available, if there is not then that's not an issue. I apologize if I was being too pushy with my comments, but I do strive for a game that can fit both your ideals and the ideals of the community.

Thank you for your reply.
